e2e/tests/metrics-reducer-view.spec.ts

```diff
@@ -11,6 +11,32 @@ test.describe('Metrics reducer view', () => {
     await metricsReducerView.assertCoreUI();
   });
 
+  test.describe('AdHoc Filters', async () => {
+    test('__name__ filter is not interpolated into the query', async ({ metricsReducerView, selectors, page }) => {
+      await metricsReducerView.goto();
+      await metricsReducerView.getByRole('combobox', { name: 'Filter by label values' }).click();
+      await metricsReducerView.getByRole('option', { name: '__name__' }).click();
+      await page.keyboard.type('=');
+      await page.keyboard.press('Enter');
+      await page.keyboard.type('grafana_database_conn_idle');
+      await page.keyboard.press('Enter');
+      await metricsReducerView.getByGrafanaSelector(selectors.components.RefreshPicker.runButtonV2).click();
+
+      await metricsReducerView.getByTestId('select-action-a_utf8_http_requests_total').click();
+
+      await metricsReducerView
+        .getByRole('button', { name: UI_TEXT.METRIC_SELECT_SCENE.SELECT_NEW_METRIC_TOOLTIP })
+        .click();
+
+      await metricsReducerView.assertCoreUI();
+      await expect(
+        metricsReducerView
+          .getByTestId('data-testid Panel header a_utf8_http_requests_total')
+          .getByTestId('data-testid Panel status error')
+      ).not.toBeVisible();
+    });
+  });
```

Comment:
This is the assertion for the panel error not being there:

<img width="264" alt="image" src="https://github.com/user-attachments/assets/21c750e3-16e2-47a8-81aa-3990cdfacb58" />

HOWEVER... for some reason this only triggers when setting playwright to record a video, not on headless mode, hence why i added `retain-on-failure` on the playwright options to force it to register the error, or in this case to force it to register the error no longer happening